app/src/androidTest/java/com/github/sdp/mediato/ProfileFragmentTest.java

```diff
@@ -76,30 +75,30 @@ public void testRecentlyWatchedTitle() {
     recentlyWatchedText.check(matches(withText("Recently watched")));
   }
 
-  // Test whether the "Add movie" button is displayed
+  /*// Test whether the "Add movie" button is displayed

```

Comment:
Is there a reason why these tests are commented out ?

Reply:
Good question